README.md

```diff
@@ -5,7 +5,8 @@
 ### Building:
 
 ```bash
-git clone https://github.com/opencontainers/runc
+git clone https://github.com/opencontainers/runc $GOPATH/src/github.com/opencontainers/runc
```

Comment:
On Tue, Jun 23, 2015 at 03:41:52PM -0700, Lei Xue wrote:
> > -git clone https://github.com/opencontainers/runc
> > +git clone https://github.com/opencontainers/runc $GOPATH/src/github.com/opencontainers/runc
> 
> @stevvooe 
> > This doesn't work for all GOPATH setups.
> 
> Could you please specify why it will not work for all `GOPATH`
> setups?

GOPATH is a delimited list of tree roots [1].  So
/path/to/a:/path/to/b is a valid GOPATH, but ‘git clone’ isn't going
to like /path/to/a:/path/to/b/src/github.com/opencontainers/runc ;).

[1]: http://golang.org/pkg/go/build/#hdr-Go_Path